### DISCUSSION TITLE
Relation between digital coin and real economy

### DISCUSSION THREAD

**Discussion by tktkowash** [JasmyCoin/JasmyCoin#11] [General]
JasmyCoin is planned to be used in the real economy. By connecting digital coin and the real economy, barriers against funding and doing businesses around the world will be removed.

**Comment by harayamada** [comment]
＊＊＊＊ Assumptions of this discussion
1) Recognition of the "real economy" 
The movement of money and goods in which goods such as services, products, and commercial materials are supplied to meet a certain demand and settlement occurs as consideration.
This is the movement of money and goods where services, products, and commercial goods are supplied to meet a certain demand and settlement occurs as consideration.
In reality, this is a simple process of paying for the benefits of products, services, and commercial goods.

(2) Recognition of the connecting between digital tokens and the real economy
Digital tokens are used as a means of payment in the flow of "services and products (benefits) ⇔ consideration.

Based on the above, it is possible to use the following

<NO.1 TOPICS: Why use digital coins instead of fiat money  (meaning notes and coins of legal tender) for settlement?

the reason is...Digital token, which are not bound by national borders or conventional economic spheres, allow anyone to make payments freely, immediately, easily, and smoothly, and can be used for transactions around the world.

I would like you to talk about the reasons from various perspectives.
★The goal is to have as many reasons as possible.

＜NO.2 TOPICS：What kind of deal is it between who and whom,in data economy?
★The goal is to identify the transaction patterns  of data exchange or deal.

The players in the transaction are individuals / companies / markets, etc.

Individuals and companies
Individuals and markets (marketplaces)
Individuals and individuals
Companies and enterprises
Companies and markets
....etc.

Addition：What kinds of businesses are involved and in what ways?
(I think there are many different kinds of businesses in the data exchange）

＜NO3 TOPICS: Digital token as a means of payment: What kind of character is appropriate?
The goal is to find the ideal form of digital coin in an economic sphere that uses data as a commercial product.

**Reply by tktkowash** [reply]
I made new threads for these three topics.

No.1: #19 Why do people use digital coins instead of fiat money for payment?
No.2: #21 Who will make a deal in data economy? And with whom?
No.3: #22 What characteristic is appropriate for digital token as a method of payment?

Please keep good discussion in each thread! 🥳
